app/controllers/member_holds_controller.rb

```diff
@@ -0,0 +1,19 @@
+class MemberHoldsController < ApplicationController
+  def create
+    if new_hold.save
+      redirect_to item_path(item), success: "Hold placed."
+    else
+      redirect_to item_path(item), error: "Something went wrong!"
```

Comment:
If there is a validation error on the new `Hold`, will the validation error(s) be included on the error message that is rendered in the view?

Reply:
It has only the item, member and creator associations which we are passing to it thru current user and item. And now there is a generic success and error messages for hold and they will be displayed once the button is clicked.

Reply:
@gregblake I don't think we would have a error we could show them that would be more specific. Really if an error happens, it it not something the did wrong. The only thing they can do is click a button.